composer.json

```diff
@@ -11,10 +11,10 @@
     ],
     "require": {
         "php": ">=5.4",
-        "ext-mongodb": "^1.2.0"
+        "ext-mongodb": ">=1.2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8"
+        "phpunit/phpunit": ">=4.8"
```

Comment:
What does this development dependency have to do with the error you shared? I realize that PHPUnit 4.8 is outdated, but we are obligated to support PHP 5.4 and 5.5, which PHPUnit 5.0 no longer supports per [this release announcement](https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-5.0.0#dropping-support-for-php-53-php-54-and-php-55).